ZOO/ZOO/Animal.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace ZOO
+{
+    public enum FavoriteFood { Grass, Carrots, Meat, Milk, Bananas, Fish, Branches, Corn }
+    public class Animal
+    {
+        private string animalName;
+        private string species;
+        private FavoriteFood favoriteFood;
+        public Zoo CurrentZoo { get; set; }
```

Comment:
zoo trebuie sa contina animale, nu un animal sa contina zoo.
a zoo has a :list of animals.
nu: un animal contains a zoo